src/components/Searchbar/InstantSearch.tsx

```diff
@@ -19,39 +26,73 @@ export const InstantSearch = () => {
 
   const onLsChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
     setSearch(e.target.value);
+    setIsSearchLoading(true);
 
     await axios
-        .get(`https://jsonplaceholder.typicode.com/albums`)
-        .then((res) => {
-          setPosts(res.data);
-        })
-        .catch((error) => {
-          if (axios.isCancel(error) || error) {
-            console.log('Could not get');
-          }
-        });
+      .get(`https://jsonplaceholder.typicode.com/albums`)
+      .then((res) => {
+        setPosts(res.data);
+        setIsSearchLoading(false);
+      })
+      .catch((error) => {
+        if (axios.isCancel(error) || error) {
+          console.log('Could not get');
+        }
+      });
   };
 
   return (
     <div className="searchModule">
-      <div className="container">
-      <input
-        value={search}
-        onChange={(e) => onLsChange(e)}
-        type="text"
-        placeholder="Почніть вводити назву..."
-      />
-
-      <ul>
-        {searchRes.map((res) => {
-          return <li key={res.id}>{res.title}</li>;
-        })}
-      </ul>
+      <div className="searchModule__input ">
+        <input
+          value={search}
+          onChange={(e) => onLsChange(e)}
+          type="text"
+          placeholder="Почніть вводити назву..."
+        />
+        {search.length > 0 &&
+          (isSearchLoading ? (
```

Comment:
there is duplicate of the list container elements, add this element inside a single wrapper in this case
<ul>
{isSearchLoading ? (<li>Loading....</li>): searchRes.map(....)}
</ul>